File: Frontend/frontendWeb/src/models/alert.ts
export class alert{
    _id:String;
    alertStatus:String;
    alertType:String;
    date:Date;
    issue:String;
    employeeId:String;
    constructor(
        _id:String,
        alertStatus:String,
        alertType:String,
        date:Date,
        issue:string,
        employeeId:String
        ){
            this._id=_id;
            this.alertStatus=alertStatus;
            this.alertType=alertType;
            this.issue=issue;
            this.date=date;
            this.employeeId=employeeId;
        }
}

File: Backend/TrashTalker/Models/Alert.cs
using System;
using TrashTalker.Models.Enumerations;

namespace TrashTalker.Models
{
    /// <summary>
    /// This class represents a Alert Entity
    /// </summary>
    public class Alert
    {
        /// <summary>
        /// Identifier
        /// </summary>
        public Guid id { get; set; }
        
        /// <summary>
        /// Issue or problem found
        /// </summary>
        public string issue { get; set; }
        
        /// <summary>
        /// Status of the alert
        /// </summary>
        public AlertStatus alertStatus { get; set; }
        
        /// <summary>
        /// Type of the alert
        /// </summary>
        public AlertType alertType { get; set; }
        
        /// <summary>
        /// Date the alert was created
        /// </summary>
        public DateTime date { get; set; }
        
        /// <summary>
        /// Responsible employee
        /// </summary>
        public virtual User employee { get; set; }
    }
}